sandbox-backend/src/main/java/com/exadel/sandbox/team5/dto/EmployeeDto.java

```diff
@@ -0,0 +1,20 @@
+package com.exadel.sandbox.team5.dto;
+
+import com.exadel.sandbox.team5.entity.BaseEntity;
+import com.exadel.sandbox.team5.entity.Role;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@EqualsAndHashCode(callSuper = true)
+public class EmployeeDto extends BaseEntity {
```

Comment:
For this PR it is ok. But in the next let's create a separate base DTO with Id field. Something like `IdentifierDto`